### PR TITLE
removing managed tenant risky user docs

### PR DESCRIPTION
**Pull request by RaajGovind**

### CODE REVIEW

**Review by idwilliams — APPROVED** [review]

**Comment by idwilliams** [comment]
@Lauragra the conflicts have been resolved. Please let us know if there are other items that need to be resolved.

**Review by Lauragra — CHANGES_REQUESTED** [review]
Please add changelog entries for the deleted APIs. Thanks.

**Comment by RaajGovind** [comment]
> Please add changelog entries for the deleted APIs. Thanks.

Done, thanks!

File: changelog/Microsoft.MultiTenantPlatform.json [hunk 1]
```diff
@@ -1,454 +1,480 @@
 {
-    "changelog": [
```
Comment (Lauragra) [inline changelog/Microsoft.MultiTenantPlatform.json:2]: I'm not sure why we have a large number of deletions and additions here. I think all we need is a ChangeList entry to mention that the resource was deleted. Thanks.

Comment (idwilliams) [reply changelog/Microsoft.MultiTenantPlatform.json:2]: @Lauragra this is a good catch. Looking through the change, I am not certain why so many items are flagged as deletions. That might have been caused by formatting update that was done in Visual Studio before the file was shared to be included in the request. I have verified the only real change is the addition of the change list entry for the entity type and navigation property.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**